### DISCUSSION TITLE
PROYECTO TERCIOS COIN ESPAÑA

### DISCUSSION THREAD

**Discussion by LOBOLUI** [smartcontractkit/full-blockchain-solidity-course-py#765] [Q&A]
Estoy interesado en crear un equipo de colaboradores y programadores para mi proyecto de block chain en bienes raices.  Podriamos hacer un llamamiento o algo similar, para ver los interesados. 
Muchas gracias. Luis Heras
@Lobolui

**Comment by cromewar** [comment]
Hola @LOBOLUI la mayoría de las personas tomando este curso son angloparlantes y no estoy seguro de que sea el mejor lugar para crear equipos, por lo cual te recomiendo revisar las comunidades de discord de: Chainlink, Moralis, Avalanche... entre otros, tienen canales en español y podrás encontrar gente para el proyecto.
También es bueno participar en hackathones, siempre hay premios y VC's financiando proyectos, un saludo.
